### ISSUE TITLE
[BUG] psutil dependency requires build tools

### ISSUE THREAD

**Issue by ptm-tm** [deloarts/pytia#23]
**Describe the bug**
im run pip install and get error
can u dont use compillers for installation?
```

      copying psutil\tests\__init__.py -> build\lib.win-amd64-cpython-311\psutil\tests
      copying psutil\tests\__main__.py -> build\lib.win-amd64-cpython-311\psutil\tests
      running build_ext
      building 'psutil._psutil_windows' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for psutil
Successfully built pytia
Failed to build psutil
ERROR: Could not build wheels for psutil, which is required to install pyproject.toml-based projects
```

**Comment by deloarts** [comment]
As stated in the output, you'll have to install the Microsoft C++ Build Tools 14.0 or greater, which are required to install **psutil 5.9.0**. I'm aware of this issue and I'll address it as soon as i push another version.

Note to me: Issue should be resolved with **psutil 5.9.3**.

**Comment by ptm-tm** [comment]
yes i know thx for note
